int8 ADD_INFORMATION=0
int8 REMOVE_INFORMATION=1
int8 SET_PARAMETERS=2
int8 ASSOCIATE=3

uint32 src
int8 action
MsgInformation[] infos
MsgParameters[] params
